### DISCUSSION TITLE
Point Orbit one

### DISCUSSION THREAD

**Discussion by mcpeterc** [homebridge-xiaomi-roborock-vacuum/homebridge-xiaomi-roborock-vacuum#950] [General]
Hi.

I have a Point Orbi tone vacuum. It uses the Smart Life app and looks very similar to other vacuums listed here. What are the chances that it will work with this plugin?

**Comment by afharo** [comment]
I cannot find it in the model in the MIOT Specs: https://home.miot-spec.com/s/point%20orbi. I'm not sure this plugin can support it.
